oqum/hazard/00a_source_typologies.tex

```diff
@@ -42,7 +44,18 @@
 
         \item \Gls{charfaultsource} - A typology of source where ruptures
         always fill the entire fault surface.
+        
+        \item \Gls{nonparametricsource} - A typology of source representing a collection of ruptures, each with their associated probabilities of 0, 1, 2 ... occurrences in the investigation time
+
+    \end{itemize}
 
+    \item Sources for representing individual earthquake ruptures (for use only with the scenario hazard and risk calculators)
```

Comment:
Here to avoid any misunderstanding I would write "Individual earthquake ruptures ..."